crazyswarm2/src/teleop.cpp

```diff
@@ -211,7 +213,8 @@ class TeleopNode : public rclcpp::Node
         twist_.linear.x = getAxis(msg, axes_.x);
         twist_.linear.y = getAxis(msg, axes_.y);
         twist_.linear.z = getAxis(msg, axes_.z);
-        twist_.angular.z = getAxis(msg, axes_.yaw);
+        // twist_.angular.z = getAxis(msg, axes_.yaw);
+        twist_.angular.z = auto_yaw_rate_; 
```

Comment:
add a condition that uses the previous code if the rate is 0, and your new code otherwise.